src/Button/Button.Component.js

```diff
@@ -33,6 +33,10 @@ const clickBtnHandler = btn => {
     Approve
```

Comment:
these examples in `Button.Component.js` may be useful, but they are unrelated to search functionality. Remove (and perhaps add back with another chore PR)

Reply:
Oops! Sorry I just submitted search in my practice branch and forgot about them. Will do! :)